### PR TITLE
Update docs for Remote Segment and Remote Translog store stats surfaced via nodes/indices/cluster stats APIs

### LINKED ISSUE #4902: [DOC] Update node stats api documentation for remote store related fields in indices segments section
**What do you want to do?**
 
- [x] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 Corresponding to -
 - https://github.com/opensearch-project/OpenSearch/pull/9168
 - https://github.com/opensearch-project/OpenSearch/pull/9454

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

### LINKED ISSUE #4903: [DOC] Update node stats api documentation for remote store related fields in indices translog section
**What do you want to do?**
 
- [x] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
Corresponding to -
 - https://github.com/opensearch-project/OpenSearch/pull/8908 [WIP]

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

### LINKED ISSUE #4904: [DOC] Update remote store stats api documentation with latest segments and translog fields
**What do you want to do?**
 
- [x] Request a change to existing documentation
- [x] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 Corresponding to -
 - https://github.com/opensearch-project/OpenSearch/pull/8718
 - https://github.com/opensearch-project/OpenSearch/pull/9454
 - https://github.com/opensearch-project/OpenSearch/pull/9263

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

### LINKED ISSUE #4954: [DOC]Modify stats api documentation to reflect addition of segment replication statistics
**What do you want to do?**
 
- [x] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.

Changes requested: 
 
 1. Under list of index properties in this table (holding list of properties for index statistics for each shard), we need to add the following rows: 
 Link: https://opensearch.org/docs/latest/api-reference/nodes-apis/nodes-stats/#indices 
 

Field | Field type | Description
-- | -- | --
segments.segment_replication | Object | Segment replication statistics for all primary shards with segment replication enabled that exist on the node.
segments.segment_replication.maxBytesBehind | long | The max number of bytes behind the primary a replica is.
segments.segment_replication.totalBytesBehind | long | The total number of bytes behind the primary the replicas are. 
segments.segment_replication.maxReplicationLag | long | The max time (in milliseconds) taken by a replica to catch up to primary. 

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
PR: https://github.com/opensearch-project/OpenSearch/pull/9709 
Issue: https://github.com/opensearch-project/OpenSearch/issues/9417

### LINKED ISSUE #4993: [DOC] Update documentation for remote segment upload backpressure settings and also add this to cluster settings
**What do you want to do?**
 
- [x] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 We have toned down on the number of remote backpressure settings. We have removed the below settings -
 - remote_store.segment.pressure.upload_bytes_moving_average_window_size
 - remote_store.segment.pressure.upload_time_moving_average_window_size
 - remote_store.segment.pressure.upload_bytes_per_sec_moving_average_window_size 

Apart from this we have introduced another cluster setting - `remote_store.moving_average_window_size` which is used to replace all the above settings that have been removed. 

Remote segment backpressure web page - https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure/
Cluster setting webpage - https://opensearch.org/docs/latest/api-reference/cluster-api/cluster-settings/ 

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

### PR DESCRIPTION
**Pull request by BhumikaSaini-Amazon**
### Description
Update docs for Remote Segment and Remote Translog store stats surfaced via nodes/indices/cluster stats APIs

### Issues Resolved
* Resolves #4902
* Resolves #4903 
* Resolves #4904
* Resolves #4993
* Resolves #4954 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _api-reference/nodes-apis/nodes-stats.md [hunk 1]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
 segments.version_map_memory_in_bytes | Integer | The total amount of memory used by all version maps, in bytes. 
 segments.fixed_bit_set_memory_in_bytes | Integer | The total amount of memory used by fixed bit sets, in bytes. Fixed bit sets are used for nested objects and join fields.
 segments.max_unsafe_auto_id_timestamp | Integer | The timestamp for the most recently retired indexing request, in milliseconds since the epoch.
+segments.remote_store | Object | Statistics about Remote Segment Store operations.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:832]:
```suggestion
segments.remote_store | Object | Statistics about remote segment store operations.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 2]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+segments.remote_store.upload | Object | Statistics about Remote Segment Store upload operations.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:833]:
```suggestion
segments.remote_store.upload | Object | Statistics about remote segment store upload operations.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 3]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+segments.remote_store.upload.total_upload_size | Object | The amount of data uploaded to Remote Segment Store.
+segments.remote_store.upload.total_upload_size.started_bytes | Integer | The number of bytes for which upload to Remote Segment Store has been started.
+segments.remote_store.upload.total_upload_size.succeeded_bytes | Integer | The number of bytes for which upload to Remote Segment Store has succeeded.
+segments.remote_store.upload.total_upload_size.failed_bytes | Integer | The number of bytes for which upload to Remote Segment Store has failed.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:837]:
```suggestion
segments.remote_store.upload.total_upload_size.failed_bytes | Integer | The number of bytes that failed to upload to the remote segment store.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 4]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+segments.remote_store.upload.refresh_size_lag | Object | Statistics about the bytes lag between the Remote Segment Store and local store.
+segments.remote_store.upload.refresh_size_lag.total_bytes | Integer | The total bytes lag between the Remote Segment Store and local store.
+segments.remote_store.upload.refresh_size_lag.max_bytes | Integer | The maximum bytes lag between the Remote Segment Store and local store.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:840]:
```suggestion
segments.remote_store.upload.refresh_size_lag.max_bytes | Integer | The maximum lag in bytes during the upload refresh between the remote segment store and the local store.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 5]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+segments.remote_store.upload.max_refresh_time_lag_in_millis | Integer | The max time duration (in milliseconds) the remote refresh is behind the local refresh.
+segments.remote_store.upload.total_time_spent_in_millis | Integer | The total time duration (in milliseconds) spent on uploads to the Remote Segment Store.
+segments.remote_store.download | Object | Statistics about Remote Segment Store download operations.
+segments.remote_store.download.total_download_size | Object | The amount of data downloaded from Remote Segment Store.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:844]:
```suggestion
segments.remote_store.download.total_download_size | Object | The amount of data downloaded from the remote segment store.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 6]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+segments.remote_store.download.total_download_size.started_bytes | Integer | The number of bytes for which download from Remote Segment Store has been started.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:845]:
```suggestion
segments.remote_store.download.total_download_size.started_bytes | Integer | The number of bytes downloaded from the remote segment store after the download starts.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 7]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+segments.remote_store.download.total_download_size.succeeded_bytes | Integer | The number of bytes for which download from Remote Segment Store has succeeded.
+segments.remote_store.download.total_download_size.failed_bytes | Integer | The number of bytes for which download from Remote Segment Store has failed.
+segments.remote_store.download.total_time_spent_in_millis | Integer | The total time duration (in milliseconds) spent on downloads from the Remote Segment Store.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:848]:
```suggestion
segments.remote_store.download.total_time_spent_in_millis | Integer | The total duration in milliseconds spent on downloads from the remote segment store.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 8]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
 segments.file_sizes | Integer | Statistics about the size of the segment files.
 translog | Object | Statistics about transaction log operations for the node.
 translog.operations | Integer | The number of translog operations.
 translog.size_in_bytes | Integer | The size of the translog, in bytes.
 translog.uncommitted_operations | Integer | The number of uncommitted translog operations.
 translog.uncommitted_size_in_bytes | Integer | The size of uncommitted translog operations, in bytes.
 translog.earliest_last_modified_age | Integer | The earliest last modified age for the translog.
+translog.remote_store | Object | Statistics about Remote Translog Store operations.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:856]:
```suggestion
translog.remote_store | Object | Statistics related to operations from the remote translog store.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 9]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+translog.remote_store.upload | Object | Statistics about Remote Translog Store upload operations.
+translog.remote_store.upload.total_uploads | Object | The number of syncs to Remote Translog Store.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:858]:
```suggestion
translog.remote_store.upload.total_uploads | Object | The number of syncs to the remote translog store.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 10]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+translog.remote_store.upload.total_uploads.started | Integer | The number of syncs to Remote Translog Store that have been started.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:859]:
```suggestion
translog.remote_store.upload.total_uploads.started | Integer | The number of upload syncs to the remote translog store that have started.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 11]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+translog.remote_store.upload.total_uploads.failed | Integer | The number of syncs to Remote Translog Store that have failed.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:860]:
```suggestion
translog.remote_store.upload.total_uploads.failed | Integer | The number of failed upload syncs to the remote translog store.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 12]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+translog.remote_store.upload.total_uploads.succeeded | Integer | The number of syncs to Remote Translog Store that have succeeded.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:861]:
```suggestion
translog.remote_store.upload.total_uploads.succeeded | Integer | The number of successful upload syncs to the remote translog store.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 13]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+translog.remote_store.upload.total_upload_size | Object | The amount of data uploaded to Remote Translog Store.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:862]:
```suggestion
translog.remote_store.upload.total_upload_size | Object | The amount of data uploaded to the remote translog store.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 14]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+translog.remote_store.upload.total_upload_size.started_bytes | Integer | The number of bytes for which upload to Remote Translog Store has been started.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:863]:
```suggestion
translog.remote_store.upload.total_upload_size.started_bytes | Integer | The number of bytes actively uploading to the remote translog store after the upload has started.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 15]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+translog.remote_store.upload.total_upload_size.failed_bytes | Integer | The number of bytes for which upload to Remote Translog Store has failed.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:864]:
```suggestion
translog.remote_store.upload.total_upload_size.failed_bytes | Integer | The number of bytes that failed to upload to the remote translog store.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 16]
```diff
@@ -792,13 +829,40 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+translog.remote_store.upload.total_upload_size.succeeded_bytes | Integer | The number of bytes for which upload to Remote Translog Store has succeeded.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:865]:
```suggestion
translog.remote_store.upload.total_upload_size.succeeded_bytes | Integer | The number of bytes successfully uploaded to the remote translog store.
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure.md [hunk 17]
```diff
@@ -18,8 +18,8 @@ Remote segment backpressure is a shard-level rejection mechanism that dynamicall
...
-- **Time lag**: The time lag is computed by comparing the timestamps of the most recent local refresh and the most recent remote store segment upload. The backpressure is activated if the time lag is greater than _K_ multiplied by the moving average of the time taken to upload new segments and metadata file after each refresh. The variance factor _K_ is configurable in `remote_store.segment.pressure.time_lag.variance_factor`. The moving window size is configurable in `remote_store.segment.pressure.upload_time_moving_average_window_size`.  
+- **Bytes lag**:  The bytes lag is computed by adding the size of all files that are present in local committed segments but not in the remote store. The backpressure is activated if the bytes lag is greater than _K_ multiplied by the moving average of the size (in bytes) of files uploaded after each refresh. The variance factor _K_ is configurable in `remote_store.segment.pressure.bytes_lag.variance_factor`. The moving window size is configurable in `remote_store.moving_average_window_size`.
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure.md:21]:
```suggestion
- **Bytes lag**:  The bytes lag is computed by adding the size of all files that are present in local committed segments but not in the remote store. Backpressure is activated if the bytes lag is greater than _K_ multiplied by the moving average of the size in, in bytes, of files uploaded after each refresh. The variance factor _K_ is configurable in `remote_store.segment.pressure.bytes_lag.variance_factor`. The moving window size is configurable through the `remote_store.moving_average_window_size` setting.
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure.md [hunk 18]
```diff
@@ -18,8 +18,8 @@ Remote segment backpressure is a shard-level rejection mechanism that dynamicall
...
+- **Bytes lag**:  The bytes lag is computed by adding the size of all files that are present in local committed segments but not in the remote store. The backpressure is activated if the bytes lag is greater than _K_ multiplied by the moving average of the size (in bytes) of files uploaded after each refresh. The variance factor _K_ is configurable in `remote_store.segment.pressure.bytes_lag.variance_factor`. The moving window size is configurable in `remote_store.moving_average_window_size`.  
+- **Time lag**: The time lag is computed by comparing the timestamps of the most recent local refresh and the most recent remote store segment upload. The backpressure is activated if the time lag is greater than _K_ multiplied by the moving average of the time taken to upload new segments and metadata file after each refresh. The variance factor _K_ is configurable in `remote_store.segment.pressure.time_lag.variance_factor`. The moving window size is configurable in `remote_store.moving_average_window_size`.
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure.md:22]:
```suggestion
- **Time lag**: The time lag is computed by comparing the timestamps of the most recent local refresh and the most recent remote store segment upload. Backpressure is activated if the time lag is greater than _K_ multiplied by the moving average of the time taken to upload new segments and metadata file after each refresh. The variance factor _K_ is configurable through `remote_store.segment.pressure.time_lag.variance_factor` setting. The moving window size is configurable through the `remote_store.moving_average_window_size` setting.  
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure.md [hunk 19]
```diff
@@ -36,13 +36,11 @@ The following table lists the settings used for activating backpressure. For thr
...
-|`remote_store.segment.pressure.upload_time_moving_average_window_size`	|Integer	|The moving average window size that is used to calculate the dynamic time lag threshold for activating remote segment backpressure. The moving average is also exposed through the [Remote Store Stats API]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api/).	Default is `20`.|
 
 The following table lists the settings used for statistics.
 
 |Setting	|Data type	|Description	|
 |:---	|:---	|:---	|
-|`remote_store.segment.pressure.upload_bytes_per_sec_moving_average_window_size`	|Integer |The moving average window size, which is exposed through [Remote Store Stats API]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api/).	Default is `20`.|
+|`remote_store.moving_average_window_size`	|Integer | The moving average window size used to calculate the rolling statistic values exposed through [Remote Store Stats API]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api/).	Default is `20`. Minimum enforced is `5`. |
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure.md:45]:
```suggestion
| `remote_store.moving_average_window_size` | Integer | The moving average window size used to calculate the rolling statistic values exposed through [Remote Store Stats API]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api/). Default is `20`. Minimum enforced is `5`. |
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

**Review by Naarcha-AWS — APPROVED** [review]

File: _tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure.md [hunk 20]
```diff
@@ -18,8 +18,8 @@ Remote segment backpressure is a shard-level rejection mechanism that dynamicall
...
-- **Time lag**: The time lag is computed by comparing the timestamps of the most recent local refresh and the most recent remote store segment upload. The backpressure is activated if the time lag is greater than _K_ multiplied by the moving average of the time taken to upload new segments and metadata file after each refresh. The variance factor _K_ is configurable in `remote_store.segment.pressure.time_lag.variance_factor`. The moving window size is configurable in `remote_store.segment.pressure.upload_time_moving_average_window_size`.  
+- **Bytes lag**:  The bytes lag is computed by adding the size of all files that are present in local committed segments but not in the remote store. Backpressure is activated if the bytes lag is greater than _K_ multiplied by the moving average of the size in, in bytes, of files uploaded after each refresh. The variance factor _K_ is configurable in `remote_store.segment.pressure.bytes_lag.variance_factor`. The moving window size is configurable through the `remote_store.moving_average_window_size` setting.
```
Comment (natebower) [inline _tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure.md:21]:
```suggestion
- **Bytes lag**: The bytes lag is calculated by adding the sizes of all the files that are present in local committed segments but not in the remote store. Backpressure is activated if the bytes lag is greater than _K_ multiplied by the moving average of the size, in bytes, of the files uploaded after each refresh. The variance factor _K_ is configurable in `remote_store.segment.pressure.bytes_lag.variance_factor`. The moving window size is configurable through the `remote_store.moving_average_window_size` setting.
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure.md [hunk 21]
```diff
@@ -18,8 +18,8 @@ Remote segment backpressure is a shard-level rejection mechanism that dynamicall
...
+- **Time lag**: The time lag is computed by comparing the timestamps of the most recent local refresh and the most recent remote store segment upload. Backpressure is activated if the time lag is greater than _K_ multiplied by the moving average of the time taken to upload new segments and metadata file after each refresh. The variance factor _K_ is configurable through `remote_store.segment.pressure.time_lag.variance_factor` setting. The moving window size is configurable through the `remote_store.moving_average_window_size` setting.
```
Comment (natebower) [inline _tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure.md:22]:
```suggestion
- **Time lag**: The time lag is calculated by comparing the timestamps of the most recent local refresh and the most recent remote store segment upload. Backpressure is activated if the time lag is greater than _K_ multiplied by the moving average of the time taken to upload new segments and metadata files after each refresh. The variance factor _K_ is configurable through the `remote_store.segment.pressure.time_lag.variance_factor` setting. The moving window size is configurable through the `remote_store.moving_average_window_size` setting.  
```

File: _tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure.md [hunk 22]
```diff
@@ -36,13 +36,11 @@ The following table lists the settings used for activating backpressure. For thr
...
-|`remote_store.segment.pressure.upload_time_moving_average_window_size`	|Integer	|The moving average window size that is used to calculate the dynamic time lag threshold for activating remote segment backpressure. The moving average is also exposed through the [Remote Store Stats API]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api/).	Default is `20`.|
 
 The following table lists the settings used for statistics.
 
 |Setting	|Data type	|Description	|
 |:---	|:---	|:---	|
-|`remote_store.segment.pressure.upload_bytes_per_sec_moving_average_window_size`	|Integer |The moving average window size, which is exposed through [Remote Store Stats API]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api/).	Default is `20`.|
+| `remote_store.moving_average_window_size` | Integer | The moving average window size used to calculate the rolling statistic values exposed through [Remote Store Stats API]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api/). Default is `20`. Minimum enforced is `5`. |
```
Comment (natebower) [inline _tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure.md:45]:
```suggestion
| `remote_store.moving_average_window_size` | Integer | The moving average window size used to calculate the rolling statistic values exposed through the [Remote Store Stats API]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api/). Default is `20`. Minimum enforced is `5`. |
```

**Review by natebower — COMMENTED** [review]
@Naarcha-AWS Please see my comments and changes and let me know if you have any questions. Thanks!

File: _api-reference/nodes-apis/nodes-stats.md [hunk 23]
```diff
@@ -792,13 +829,44 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
+segments.remote_store.upload.total_upload_size.failed_bytes | Integer | The number of bytes that failed to upload to the remote segment store.
+segments.remote_store.upload.refresh_size_lag | Object | The amount of lag during upload between the remote segment store and the local store.
+segments.remote_store.upload.refresh_size_lag.total_bytes | Integer | The total bytes that lagged during the upload refresh between the remote segment store and the local store.
+segments.remote_store.upload.refresh_size_lag.max_bytes | Integer | The maximum lag in bytes during the upload refresh between the remote segment store and the local store.
+segments.remote_store.upload.max_refresh_time_lag_in_millis | Integer | The maximum duration in milliseconds the remote refresh is behind the local refresh.
+segments.remote_store.upload.total_time_spent_in_millis | Integer | The total time in milliseconds spent on uploads to the remote segment store.
+segments.remote_store.download | Object | Statistics related to download operation to the remote segment store.
+segments.remote_store.download.total_download_size | Object | The amount of data downloaded from the remote segment store.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:848]:
```suggestion
segments.remote_store.download.total_download_size | Object | The total amount of data download from the remote segment store.
```

File: _api-reference/nodes-apis/nodes-stats.md [hunk 24]
```diff
@@ -792,13 +829,44 @@ segments.index_writer_memory_in_bytes | Integer | The total amount of memory use
...
 translog.uncommitted_operations | Integer | The number of uncommitted translog operations.
 translog.uncommitted_size_in_bytes | Integer | The size of uncommitted translog operations, in bytes.
 translog.earliest_last_modified_age | Integer | The earliest last modified age for the translog.
+translog.remote_store | Object | Statistics related to operations from the remote translog store.
+translog.remote_store.upload | Object | Statistics about upload operation to the remote translog store.
+translog.remote_store.upload.total_uploads | Object | The number of syncs to the remote translog store.
+translog.remote_store.upload.total_uploads.started | Integer | The number of upload syncs to the remote translog store that have started.
+translog.remote_store.upload.total_uploads.failed | Integer | The number of failed upload syncs to the remote translog store.
+translog.remote_store.upload.total_uploads.succeeded | Integer | The number of successful upload syncs to the remote translog store.
+translog.remote_store.upload.total_upload_size | Object | The amount of data uploaded to the remote translog store.
```
Comment (Naarcha-AWS) [inline _api-reference/nodes-apis/nodes-stats.md:866]:
```suggestion
translog.remote_store.upload.total_upload_size | Object | The total amount of data uploaded to the remote translog store.
```

**Pull request merged**